api_app/tests_ma/test_api/test_routes/test_airlock_resource_helpers.py

```diff
@@ -0,0 +1,222 @@
+from models.domain.airlock_review import AirlockReview, AirlockReviewDecision
+from models.domain.airlock_resource import AirlockResourceType
+from models.domain.airlock_request import AirlockRequest, AirlockRequestStatus, AirlockRequestType
+from azure.eventgrid import EventGridEvent
+
+pytestmark = pytest.mark.asyncio
+
+
+WORKSPACE_ID = "abc000d3-82da-4bfc-b6e9-9a7853ef753e"
+AIRLOCK_REQUEST_ID = "5dbc15ae-40e1-49a5-834b-595f59d626b7"
+AIRLOCK_REVIEW_ID = "96d909c5-e913-4c05-ae53-668a702ba2e5"
+
+
+@pytest.fixture
+def airlock_request_repo_mock():
+    with patch('azure.cosmos.CosmosClient') as cosmos_client_mock:
+        yield AirlockRequestRepository(cosmos_client_mock)
+
+
+@pytest.fixture
+def airlock_review_repo_mock():
+    with patch('azure.cosmos.CosmosClient') as cosmos_client_mock:
+        yield AirlockReviewRepository(cosmos_client_mock)
+
+
+def sample_airlock_request():
+    airlock_request = AirlockRequest(
+        id=AIRLOCK_REQUEST_ID,
+        resourceType=AirlockResourceType.AirlockRequest,
+        workspaceId=WORKSPACE_ID,
+        requestType=AirlockRequestType.Import,
+        files=[],
+        businessJustification="some test reason"
+    )
+    return airlock_request
+
+
+def sample_airlock_request_updated():
+    airlock_request = AirlockRequest(
+        id=AIRLOCK_REQUEST_ID,
+        resourceType=AirlockResourceType.AirlockRequest,
+        workspaceId=WORKSPACE_ID,
+        requestType=AirlockRequestType.Import,
+        files=[],
+        businessJustification="some test reason",
+        status=AirlockRequestStatus.Submitted
+    )
+    return airlock_request
+
+
+def sample_status_changed_event_after_update():
+    status_changed_event = EventGridEvent(
+        event_type="statusChanged",
+        data={
+            "request_id": AIRLOCK_REQUEST_ID,
+            "status": "submitted",
+            "type": AirlockRequestType.Import,
+            "workspace_id": WORKSPACE_ID[-4:]
+        },
+        subject=f"{AIRLOCK_REQUEST_ID}/statusChanged",
+        data_version="2.0"
+    )
+    return status_changed_event
+
+
+def sample_status_changed_event():
+    status_changed_event = EventGridEvent(
+        event_type="statusChanged",
+        data={
+            "request_id": AIRLOCK_REQUEST_ID,
+            "status": "draft",
+            "type": AirlockRequestType.Import,
+            "workspace_id": WORKSPACE_ID[-4:]
+        },
+        subject=f"{AIRLOCK_REQUEST_ID}/statusChanged",
+        data_version="2.0"
+    )
+    return status_changed_event
+
+
+def sample_airlock_review():
+    airlock_review = AirlockReview(
+        id=AIRLOCK_REVIEW_ID,
+        resourceType=AirlockResourceType.AirlockReview,
+        workspaceId=WORKSPACE_ID,
+        requestId=AIRLOCK_REQUEST_ID,
+        reviewDecision=AirlockReviewDecision.Approved,
+        decisionExplanation="test explaination"
+    )
+    return airlock_review
+
+
+class TestAirlockResourceHelpers:
```

Comment:
I think that defining it under a class is redundant?

Reply:
Done